### ISSUE TITLE
Merged too soon

### ISSUE THREAD

**Issue by gracekabaghe** [Andy-E-mc2/website#2]
Glad you figured out merging, however, you merged too soon. 💯 
- A pull request needs to be approved before it's merged into the main branch.
--so when a project has no open PR, an [issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) can be raised -- 
Once you're done reading, feel free to close this issue.
**Next steps**
Create a new branch (git checkout -b <name>)
- Add 2 more content to your home page e.g 2-3 paragraphs (ok to use placeholder text like [lorem ipsum](https://www.lipsum.com/)
- Add a footer section to your home page
- Push and create a pull request.

**Comment by gracekabaghe** [comment]
- before creating a new branch, pull the code from remote to local by running **git pull** in terminal, then create a new branch.
